src/main/scala/org/scalajs/tools/tsimporter/Trees.scala

```diff
@@ -26,6 +26,14 @@ object Trees {
   sealed trait TypeTree extends Tree
   sealed trait MemberTree extends Tree
 
+  sealed trait Modifier
```

Comment:
Prefer an `abstract sealed class`, like `Tree` above.

(also add a blank line between the class/trait and the object below)